spec/models/user_spec.rb

```diff
@@ -3,4 +3,18 @@
 RSpec.describe User, type: :model do
   it { is_expected.to be }
 
+  describe '#authenticate' do
+
+    it 'returns true if given correct password' do
+      user = User.create(first_name: "Elishka", last_name: "Flint", email: "elishka@keepingitrails.com", password: "pa55w0rd")
```

Comment:
Would it be possible to replace the details in the tests with like first_name: "test_first_name" etc rather than my details, might look a little more professional that way

Reply:
(i know we've done the same in ours, will change there too on the next branch)

Reply:
same in capybara tests above ^^^

Reply:
Good shout, updated web_helper file too